cubic-server/Client.cpp

```diff
@@ -293,6 +293,16 @@ void Client::_onEncryptionResponse(const std::shared_ptr<protocol::EncryptionRes
     _log->debug("Got a Encryption Response");
 }
 
+void Client::disconnectClient(chat::Message &message)
+{
+    nlohmann::json json;
+    
+    json["translate"] = "chat.type.text";
+    json["with"] = nlohmann::json::array({{"text", "PlayerName"}, {message.toJson()}});
```

Comment:
Please go on multiple lines here

Reply:
That's probably my fault, this is supposed to be in the Message class...